fastseq_cli/transformers_generate.py

```diff
@@ -70,10 +81,17 @@ def __init__(self, msg_queue, fout):
         self.waiting_for=0
         self.dec_buf = {}
 
-    def process_dec(self, dec):
-        for hypothesis in dec:
+    def process_dec(self, dec_scores):
```

Comment:
could you add py doc for this function to describe what it is for and inputs (`dec_scores`) looks like. This will make the code easy to read.